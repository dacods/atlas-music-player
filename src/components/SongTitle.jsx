function SongTitle() {
    return (
        <div>
            <h1 className="text-2xl font-bold " >Tidal Drift</h1>
            <p className="text-base font-normal text-[#64748B]">Echoes of the Sea</p>
        </div>
    )
}

export default SongTitle;