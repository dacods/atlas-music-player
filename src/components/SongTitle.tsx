interface SongTitleProps {
  song?: {
    title: string;
    artist: string;
  };
}

function SongTitle({ song }: SongTitleProps) {
    return (
        <div>
            <h1 className="text-2xl font-bold " >{song?.title}</h1>
            <p className="text-base font-normal text-charcoal-gray">{song?.artist}</p>
        </div>
    )
}

export default SongTitle;