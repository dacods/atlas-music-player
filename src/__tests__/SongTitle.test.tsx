import { test, expect } from 'vitest';
import { render } from '@testing-library/react';
import SongTitle from '../components/SongTitle';

test('render SongTitle', () => {
    const song = { title: "Song 1", artist: "Artist 1" };
    const { container } = render(<SongTitle song={song}></SongTitle>);
    expect(container).toMatchSnapshot();
});