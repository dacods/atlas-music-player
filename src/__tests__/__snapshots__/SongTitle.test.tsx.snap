// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render SongTitle 1`] = `
<div>
  <div>
    <h1
      class="text-2xl font-bold "
    >
      Song 1
    </h1>
    <p
      class="text-base font-normal text-charcoal-gray"
    >
      Artist 1
    </p>
  </div>
</div>
`;
